WooCommerce/Classes/ViewRelated/Blaze/CampaignCreation/BlazeCampaignCreationForm.swift

```diff
@@ -304,16 +304,17 @@ private extension BlazeCampaignCreationForm {
                     }
                 }
 
-                if !viewModel.isLoadingAISuggestions,
+                if viewModel.isLoadingAISuggestions ||
                    viewModel.ctaText.isNotEmpty {
```

Comment:
Super nit: I think the indentation needs to be updated here.

Reply:
Good eye, fixed in e291f45